.github/workflows/squash.yml

```diff
@@ -10,11 +10,11 @@ on:
 
 jobs:
   build-docker-24:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13.0-rc.1']
```

Comment:
The readme still states it supports Python 3.6 and above. Note that CEKit uses docker-squash and supports 3.6 and above ( https://github.com/cekit/cekit/blob/develop/.github/workflows/cekit.yml#L20 ). Why not keep the 3.6/3.7 if its not breaking anything and just add the newer versions?

Reply:
I can restore it, it was just pointed by a user that these are EOLed already.

Reply:
Yeah but potentially not EOL in RHEL or all platforms CEKit is used on?